strapi/strapi
# feat(content-releases): publish a release

### What does it do?

Add endpoint for publish a release. For the moment, publish action is only available for superadmin users. 

### How to test it?

You should have a strapi project with content releases enabled and logged in with a superadmin user.
1. Make a POST request to `/content-releases/:id/publish` endpoint
2. If everything is okay, it should return the release with the new publishedAt field and all the entries related should be published/unpublished
